src/main/java/lotto/controller/LottosBundleGenerator.java

```diff
@@ -0,0 +1,34 @@
+package lotto.controller;
+
+import lotto.domain.Lotto;
+import lotto.domain.LottoCount;
+import lotto.domain.LottoPrice;
+
+import java.util.List;
+
+public class LottosBundleGenerator implements LottoGenerator {
+    private final List<LottoGenerator> lottosGenerators;
+
+    public LottosBundleGenerator(LottoPrice price, LottoCount manualCount, List<String> manualLottoText) {
```

Comment:
```suggestion
    public LottosBundleGenerator(LottoPrice price, List<String> manualLottoText) {
```
manualLottoText 크기가 manualCount인데 manualCount는 필요없는 값이지 않을까?